### ISSUE TITLE
Vincenty results are wrong

### ISSUE THREAD

**Issue by eliecharra** [mjaschen/phpgeo#42]
It seems that Vincenty calculation are wrong.

Distance between theses two points return `0`, it should return `1.255` meters

```php
<?php
var_dump((new Vincenty())
    ->getDistance(
        new Coordinate(45.306833,5.887717), 
        new Coordinate(45.306833,5.887733)
)); // float(0)
```

Using some other lib, it return the correct value.

* Using http://www.cqsrg.org/tools/GCDistance/

![image](https://user-images.githubusercontent.com/6154987/50407197-adaf8a80-07d2-11e9-8194-39640ac0d3dc.png)

* Using https://pypi.org/project/vincenty/

```
>>> point1 = (45.306833,5.887717)
>>> point2 = (45.306833,5.887733)
>>> vincenty(point1, point2) * 1000
1.2550000000000001
```

**Comment by mjaschen** [comment]
Thanks for the report, the fixed now (in version 2.0.4).

**Comment by eliecharra** [comment]
:+1: Awesome reactivity, thanks for your work !

a2b4f8bd735610a3df351a5bd79109f81c800889 Fix the issue for every point of my dataset :champagne:
